apps/src/code-studio/pd/application/teacher/ProfessionalLearningProgramRequirements.jsx

```diff
@@ -21,6 +26,19 @@ import {LabeledDynamicCheckBoxesWithAdditionalTextFields} from '../../form_compo
 import {useRegionalPartner} from '../../components/useRegionalPartner';
 import {FormContext} from '../../form_components_func/FormComponent';
 
+const getProgramName = program => {
```

Comment:
I could be wrong, but I *think* this info is in `data.program.name` (I'm looking here https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/code-studio/pd/application/teacher/ChooseYourProgram.jsx#L163)

Reply:
programInfo.name is defined very similarly in that file (lines 25-36) since the name defined in data is a much longer title (e.g. "Computer Science A (appropriate for 10th - 12th grade, and can be implemented as an AP or non-AP introductory Java programming course)")

Reply:
Ahh, I missed the source of truth! Good call 👍 . Could this stuff go in `ApplicationConstants.jsx`? Not a huge deal.

Reply:
I think that's a good idea! I think I'm going to throw together a quick PR to make that change since it will involve a couple changes and it will be nice to have the consistency of this being the text PR and then the next one being a shift in Constants definition.

Reply:
Okay –– no worries! Thanks for trying